src/wasm-builder.h

```diff
@@ -321,15 +321,17 @@ class Builder {
 
   static Index addVar(Function* func, Name name, WasmType type) {
     // always ok to add a var, it does not affect other indices
+    assert(name.is());
```

Comment:
Pleaset add another assert here that the name is unique (not already in `func->localIndices`).